src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -180,7 +181,9 @@ const FormattedReport = ({ summaries, weekIndex, role }) => {
     return (
       <div>
         <b>Bio announcement:</b>
-        {bioPosted ? ' Posted' : ' Requested'}
+        {bioPosted === 'default' ? 'Not requested/posted' :
+         bioPosted === 'posted' ? 'Posted' : 
+         'Requested'}
```

Comment:
```suggestion
        {bioPosted === 'default' ? ' Not requested/posted' :
         bioPosted === 'posted' ? ' Posted' : 
         ' Requested'}
```
Still missing spaces.